specification/iotoperationsorchestrator/IoTOperationsOrchestrator.Management/instance.tsp

```diff
@@ -58,6 +61,15 @@ model TargetSelectorProperties {
   name?: string;
 }
 
+@doc("Defines parameter properties.")
+model ParamProperties {
+  @doc("Additional properties.")
+  additionalProperties?: string[];
+
+  @doc("Property type.")
+  type: string;
+}
+
```

Comment:
@saileerane I'm not certain that this object definition is accurate. `parameters` as defined in orchestrator schema is defined as as map of unknown string keys to strings.

This schema as defined here states that it's a known object with 2 keys. `additionalProperties` and `type`

Reply:
I see. How do I translate map[string]string here to define the parameters?

Reply:
you can set parameters to `Record<unknown>` i.e
```js
model InsanceProperties {
...
  parameters?: Record<unknown>
}
```

Reply:
Record<unknown> was giving a compilation error on running this command  npx tsv specification/iotoperationsorchestrator/IoTOperationsOrchestrator.Management. Had to modify it to Record<string>
![image](https://github.com/Azure/azure-rest-api-specs/assets/109764127/09ef8675-2acb-4afc-9230-af8487bc5664)